src/xsplit/XsplitWrap.js

```diff
@@ -62,7 +62,9 @@ class XsplitWrapper extends SharedWrapMixin {
 
     return (
       <div className="App" style={{ width: '1920px', height: '1080px', overflow: 'hidden' }}>
-        { matchStats && setStats && config && theme && configLoaded &&
+        { !(this.props.shouldFetchMatch && !matchStats)
```

Comment:
After looking at this a few times I'm wondering if maybe you should bundle this into a utility/helper function for readability. Maybe wrap this logic into a `ShouldAndCanFetchMatchSetStats()`?

Reply:
Even demorgansing this would make it more clear. made an issue